front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftComponents/OpenToParticipation/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { Multiloc } from 'typings';
+
+import useProjectsWithActiveParticipatoryPhase from 'api/projects_mini/useProjectsWithActiveParticipatoryPhase';
+
+import useLocalize from 'hooks/useLocalize';
+
+import messages from './messages';
+import ProjectCarrousel from './ProjectCarrousel';
+import Settings from './Settings';
+
+interface Props {
+  titleMultiloc: Multiloc;
+}
+
+const OpenToParticipation = ({ titleMultiloc }: Props) => {
+  const localize = useLocalize();
+  const { data, hasNextPage, fetchNextPage } =
+    useProjectsWithActiveParticipatoryPhase();
+  const projects = data?.pages.map((page) => page.data).flat();
+
+  if (!projects) return null;
+  if (projects.length === 0) return null;
```

Comment:
Could you use !!projects instead here and cover both cases?

Reply:
Unfortunately not, as empty arrays are seen as truthy:

```
!![] // true
```